pkg/operator/ceph/cluster/controller.go

```diff
@@ -327,10 +327,22 @@ func (r *ReconcileCephCluster) reconcileDelete(cephCluster *cephv1.CephCluster)
 		}
 	}
 
-	// Remove finalizer
-	err = removeFinalizer(r.client, nsName)
+	// Remove cephcluster finalizer
```

Comment:
Let's have a single function that removes all the finalizers.

Reply:
Nice point, I was thinking of it.

But with different resources how are you thinking to have it?

```	
	monSecret := &corev1.Secret{}
	cephCluster := &cephv1.CephCluster{}
	monConfigMap := &corev1.ConfigMap{}
```
We need to Get the resources from the client first.

Reply:
What about just a function that take a client as a parameter and then run all three functions:

* `removeCephClusterFinalizer`
* `removeMonSecretFinalizer`
* `removeMonConfigMapFinalizer` 

in it?

Reply:
yup:)